### PR TITLE
Add email play back and signout option to Digital Pack checkout

### PR DESCRIPTION
**Pull request by walaura**
## Why are you doing this?

Add a 'ghost' email field to the checkout page and two links to get help changing the feld or signing out

[**Trello Card**](https://trello.com/c/vz1dS1Vb/2091-add-email-play-back-and-signout-option-to-digital-pack-checkout)

## Screenshots

screenshot 2019-01-22 at 1 47 40 pm

screenshot 2019-01-22 at 1 47 43 pm

### CODE REVIEW

File: assets/components/checkoutExpander/checkoutExpander.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.component-checkout-expander__strong {
+  text-decoration: underline;
+  font-weight: 600;
+}
+
+.component-checkout-expander__expando {
+  margin-bottom: $gu-v-spacing;
+  button {
```
Comment (akash1810) [inline assets/components/checkoutExpander/checkoutExpander.scss]: do we prefer styling classes over elements?

File: assets/pages/digital-subscription-checkout/components/checkoutForm.jsx [hunk 2]
```diff
@@ -131,6 +135,25 @@ function CheckoutForm(props: PropTypes) {
...
+          disabled
+          value={props.email}
+          footer={(
+            <span>
+              <CheckoutExpander copy="Want to use a different email address?">
+                <p>You will be able to edit this in your account once you have completed this checkout.</p>
+              </CheckoutExpander>
+              <CheckoutExpander copy="Not you?">
+                <p>
+                  <Button appearance="greyHollow" icon={null} type="button" aria-label={null} onClick={() => props.signOut()}>Sign out</Button> and create a new account.
```
Comment (akash1810) [inline assets/pages/digital-subscription-checkout/components/checkoutForm.jsx]: Could `icon={null}` be the default behaviour of `Button`?

Comment (walaura) [reply assets/pages/digital-subscription-checkout/components/checkoutForm.jsx]: it's confusing to read, so many of our button instances just have the arrow in there that at the time it felt like a nice syntax shortcut. Now that contributions is using this button too it's a very good point to raise though!

File: assets/components/checkoutExpander/checkoutExpander.scss (hunk 1 above)
Comment (walaura) [reply assets/components/checkoutExpander/checkoutExpander.scss]: we do! this PR is a bit half baked and it shows 😆

Comment (walaura) [inline assets/components/checkoutExpander/checkoutExpander.scss]:
```suggestion
  . component-button {
```

**Pull request merged**